critiquebrainz/db/revision.py

```diff
@@ -148,23 +152,34 @@ def get_revision_number(review_id, revision_id):
     return rev_num
 
 
-def create(review_id, text):
+def create(review_id, text, rating):
```

Comment:
Did you forget to check that either `text` or `rating` is passed to this function?